doc/classes/OS.xml

```diff
@@ -426,7 +426,7 @@
 			<description>
 				Returns an array of the system substitute font file paths, which are similar to the font with [param font_name] and style for the specified text, locale and script. Returns empty array if no matching fonts found.
 				The following aliases can be used to request default fonts: "sans-serif", "serif", "monospace", "cursive", and "fantasy".
-				[b]Note:[/b] Depending on OS, it's not guaranteed that any of the returned fonts is suitable for rendering specified text. Fonts should be loaded and checked in the order they are returned, and the first suitable one used.
+				[b]Note:[/b] Depending on OS, it's not guaranteed that any of the returned fonts are suitable for rendering specified text. Fonts should be loaded and checked in the order they are returned, and the first suitable one used.
```

Comment:
I'd suggest "will be suitable", to me it flows better and the tense fits better

Reply:
And from what I can find both "is" and "are" are valid here, and carries slightly different meaning, as this talks about that it doesn't guarantee the validity of any of the fonts I would argue "is" to be more correct, but neither is necessarily incorrect

Reply:
Thanks for the feedback, I appreciate it. I wholeheartedly agree with the suggestion, I'll update it soon. I was thinking it myself but decided to keep it simple. Which leads to your next point...

As far as "is" vs "are": whoops. I didn't realize that "any of" is different than "any". Nice catch. I can't believe I created a typo trying to fix one. I'll remember for future reference.

Reply:
Yeah I agree. Neither is necessarily wrong, it's just that the singular is considered more appropriate in a formal context (according to the below link) and thus I was wrong to change it.

https://english.stackexchange.com/questions/475028/is-any-of-them-used-with-a-singular-or-plural-verb

Regardless, saying "will be" sounds better anyway and so I've updated it.